invenio_app_rdm/cli.py

```diff
@@ -93,3 +94,80 @@ def rebuild_all_indices(order):
             else:
                 # success
                 click.secho("Done.", fg="green")
+
+
+@cli_users.command("delete")
+@click.argument("user_id")
+def user_delete(user_id):
+    """Delete a user, checking first the existence of RDM records."""
+    from invenio_accounts.proxies import current_datastore
+
+    user = current_datastore.get_user(user_id)
+    if not user:
+        click.secho(f"User {user_id} not found.", fg="red")
+        return
+
+    if not click.confirm(
+        f'Are you sure you want to delete user "{user.id}/{user.username}/{user.email}"?'
+    ):
+        click.secho("Aborted.", fg="red")
+        return
+
+    from invenio_access.utils import get_identity
+
+    idty = get_identity(user)
+
+    from invenio_rdm_records.proxies import current_rdm_records_service
+
+    drafts = current_rdm_records_service.search_drafts(idty)
```

Comment:
why is it not possible to delete a user if drafts are still active? in my opinion it should not matter where a user is registered or part of, if the user wants that their personal data should be deleted it should be deleted. the discussion is how to handle that situation in a way that it doesn't crashes the system